packages/flutter/lib/src/material/toggle_buttons.dart

```diff
@@ -1550,3 +1575,134 @@ class _SelectToggleButtonRenderObject extends RenderShiftedBox {
     }
   }
 }
+
+/// A widget to pad the area around a [ToggleButtons]'s children.
```

Comment:
I believe this code was based on a similar class in ButtonStyleButton.dart. It would be helpful to leave a private comment about that here, in case a forthcoming change to this class might also benefit the other version.